Content.Shared/Cuffs/SharedCuffableSystem.cs

```diff
@@ -471,6 +471,9 @@ public bool TryAddNewCuffs(EntityUid target, EntityUid user, EntityUid handcuff,
             if (!_interaction.InRangeUnobstructed(handcuff, target))
                 return false;
 
+            cuff.Used = true;
```

Comment:
im just gonna put here that I concern over this being BEFORE the early return ahead of it, but apparently it was here before so im just gonna roll with it for now